inc/class-wpseo-replace-var-data.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Internals
+ * @since   7.0
+ */
+
+/**
+ * Class WPSEO_Replace_Var_Data
+ *
+ * This class is used by class-wpseo-replace-vars to store the data of a single row of the snippet variables table.
+ */
+class WPSEO_Replace_Var_Data {
+
+	/**
+	 * @var string  The label, or niceName, of the replacement variable.
+	 */
+	protected $label;
+
+	/**
+	 * @var string  The description of the replacement variable.
+	 */
+	protected $description;
+
+	/**
+	 * WPSEO_Replace_Var_Data constructor.
+	 *
+	 * @param string $label         The label, or niceName, of the replacement variable.
+	 * @param string $description   The description of the replacement variable.
```

Comment:
Can you align the comments nicely. Now there are too many spaces before `The descreiption`